Source/EasyNetQ.ApprovalTests/EasyNetQ.approved.txt

```diff
@@ -345,21 +345,27 @@ namespace EasyNetQ
     }
     public interface IAdvancedBus
     {
+        [System.Obsolete("IsConnected is deprecated because it is misleading. Please use GetConnectionStatu" +
```

Comment:
does plus sign really necessary here?

Reply:
It is autogenerated, please ignore this file.